### DISCUSSION TITLE
How to open the .spc files in raw_output_proffast？

### DISCUSSION THREAD

**Discussion by zyzyzzy2qx** [tum-esm/em27-retrieval-pipeline#110] [Q&A] [answered]
Hi everyone, I have been troubled by a problem for a long time that is How to open the .spc files in raw_output_proffast？

**Comment by dostuffthatmatters** [comment, marked as answer]
Hi @zyzyzzy2qx,

I am not sure how to open them. The best place to ask this is the Repository of the Proffast Pylot (https://gitlab.eudat.eu/coccon-kit/proffastpylot). If you find out how, I am happy to add it to the documentation of this pipeline.

**EDIT: this issue is tracked here https://github.com/coccon/proffastpylot/issues/11**

Best,
Moritz

PS: Which institute/research group are you from?

**Reply by zyzyzzy2qx** [reply]
@dostuffthatmatters OK, I'll go find it out, thank you~

**Reply by zyzyzzy2qx** [reply]
@dostuffthatmatters Hi, I'm a postdoctoral from Aerospace Information Research Institute, Chinese Academy of Sciences. By the way, the interferogram data you post in the /data/testing/inputs/ cannot be achieved, can you please e-mail the interferogram data to me, my e-mail address is zhaoyang_0320@163.com? Thank you very much.
PS: Is there any way to download interferogram data of EM27/SUN, I only found the retrieved results on the COCCON website, but I want the interferogram data. (I'm new to ground-based remote sensing, my tutor insistently asked me to find the EM27/SUN raw data before our equipment is in place, which has troubled me a lot recently lol.) Thank you very much, I hope this won't bother you.

**Reply by dostuffthatmatters** [reply]
Hi @zyzyzzy2qx,

Great to hear that you are joining the EM27/SUN fleet!

The interferogram data in `data/testing/inputs` will be downloaded automatically when running the tests, because they are too big for the GitHub repository. It is available at https://syncandshare.lrz.de/dl/fiEh5KjXeb8b715kU4cwWi/em27-retrieval-pipeline-test-interferograms-2024-09-17.tar.gz

I don't think anyone publishes their EM27/SUN interferograms because they are too big (up to 6 GB per instrument per day). And also because the raw data can be used for more than the XCO2/XCH4/XCO time series and always has to be calibrated first.

Best,
Moritz

**Reply by zyzyzzy2qx** [reply]
@dostuffthatmatters Thank you very much!!! Your interferogram data helped me a lot! Meanwhile, thanks for explaining the difficulty of uploading EM27/SUN raw data. I will keep going and hope to learn more from you~

**Reply by zyzyzzy2qx** [reply]
Hi @dostuffthatmatters , do you know how to plot the spectral fits for each micro window of the inversion results?

**Reply by dostuffthatmatters** [reply]
@zyzyzzy2qx No, sorry.
